worlds/empty_with_plugins.sdf

```diff
@@ -6,6 +6,23 @@
       <real_time_factor>1.0</real_time_factor>
     </physics>
 
+    <plugin filename="ignition-gazebo-contact-system" name="gz::sim::systems::Contact" />
+    <plugin filename="ignition-gazebo-imu-system" name="gz::sim::systems::Imu" />
+    <plugin filename="ignition-gazebo-physics-system" name="gz::sim::systems::Physics" />
+    <plugin filename="ignition-gazebo-scene-broadcaster-system"
+      name="gz::sim::systems::SceneBroadcaster" />
+    <plugin filename="ignition-gazebo-sensors-system" name="gz::sim::systems::Sensors" />
+    <plugin filename="ignition-gazebo-user-commands-system" name="gz::sim::systems::UserCommands" />
+
+    <spherical_coordinates>
+      <surface_model>EARTH_WGS84</surface_model>
+      <world_frame_orientation>ENU</world_frame_orientation>
+      <latitude_deg>50.088384</latitude_deg>
+      <longitude_deg>19.939128</longitude_deg>
+      <elevation>0</elevation>
+      <heading_deg>0</heading_deg>
+    </spherical_coordinates>
```

Comment:
APPLIES TO ALL WORLD FILES

How about creating a separate SDF with plugins to avoid defining three times the same values? We can then include plugins in world files.

Reply:
Godd idea, but I do not know if it's possible
![image](https://github.com/husarion/husarion_gz_worlds/assets/126687345/d7cf0e18-39ef-41e2-83d9-e83b5acf97d9)